src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -24,7 +24,7 @@
                 responsive="sm"
                 show-empty>
                 <template v-for="(_, slot) in $scopedSlots" v-slot:[slot]="props">
-                    <slot :name="slot" v-bind="props" />-
+                    <slot :name="slot" v-bind="props" />
```

Comment:
This was the culprit responsible for the rogue additional '-' lines... :facepalm: